### PR TITLE
v24.2.0-alpha.1 release notes

### PR DESCRIPTION
**Pull request by kathancox**
Fixes REL-1077

### CODE REVIEW

File: src/current/_data/versions.csv [hunk 1]
```diff
@@ -14,3 +14,4 @@ v22.2,2022-12-05,2023-12-05,2024-06-05,N/A,N/A,N/A,N/A,N/A,v22.1,release-22.2
 v23.1,2023-05-15,2024-05-15,2024-11-15,23.1.11,23.1.12,2023-11-13,2024-11-13,2025-11-13,v22.2,release-23.1
 v23.2,2024-02-05,2025-02-05,2025-08-05,N/A,N/A,N/A,N/A,N/A,v23.1,release-23.2
 v24.1,2024-05-20,2025-05-20,2025-11-20,N/A,N/A,N/A,N/A,N/A,v23.2,release-24.1
+v24.2,N/A,N/A,N/A,v24.1,master
```
Comment (mdlinville) [inline src/current/_data/versions.csv:17]:
```suggestion
v24.1,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,master
```

File: src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+- Added an error message for [changefeed]({% link v24.1/change-data-capture-overview.md %}) options and parameters that are not supported by the [Apache Pulsar sink]({% link v24.1/changefeed-sinks.md %}#apache-pulsar). [#124581][#124581]
+- [Scheduled changefeeds]({% link v24.1/create-schedule-for-changefeed.md %}) now pause after being [restored]({% link v24.1/restore.md %}) onto a different cluster, and after completion of [physical cluster replication]({% link v24.1/physical-cluster-replication-overview.md %}) to avoid inadvertent concurrent execution of the same schedule on multiple clusters. [#124631][#124631]
+- Added a new metric to facilitate the new [changefeed]({% link v24.1/change-data-capture-overview.md %}) billing strategy for usage in CockroachDB {{ site.data.products.cloud }} clusters. [#123145][#123145]
+- The `DEBUG_PAUSE_ON` option has been removed and replaced with the `restore.restore_after_failure` pause point to match other pause points used throughout CockroachDB. You can set this pause point by running: `SET CLUSTER SETTING jobs.debug.pausepoints = 'restore.after_restore_failure'`. [#125158][#125158]
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:33]: do we even have any docs on this "pause point" stuff?

is it related to https://www.cockroachlabs.com/docs/stable/pause-job ?

File: src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+- Multiple or large numbers of [grants]({% link v24.1/show-grants.md %}) on tables and types within one [transaction]({% link v24.1/transactions.md %}) now run faster. [#123414][#123414]
+- [Lock operations]({% link v24.1/architecture/transaction-layer.md %}#writes-and-reads-phase-1) are now removed from [query plans]({% link v24.1/cost-based-optimizer.md %}) when the optimizer can prove that no rows would be locked. [#124830][#124830]
+- Some privilege checks when scanning the [`crdb_internal.system_jobs`]({% link v24.1/crdb-internal.md %}) internal table now happen once before the scan, instead of once for each row. This will improve performance for queries that read from `crdb_internal.system_jobs`. [#117438][#117438]
+- Improved the initial range descriptor scan on startup. [Node startup]({% link v24.1/cockroach-start.md %}) should now be faster on larger stores [#123959][#123959]
+
+<h3 id="v24-2-0-alpha-1-contributors">Contributors</h3>
+
+This release includes 1110 merged PRs by 91 authors.
+We would like to thank the following contributors from the CockroachDB community:
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:102]: Lines 102-113 should be removed

From "we would like ..." to the end of the bulleted list of folks

File: src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+- Multiple or large numbers of [grants]({% link v24.1/show-grants.md %}) on tables and types within one [transaction]({% link v24.1/transactions.md %}) now run faster. [#123414][#123414]
+- [Lock operations]({% link v24.1/architecture/transaction-layer.md %}#writes-and-reads-phase-1) are now removed from [query plans]({% link v24.1/cost-based-optimizer.md %}) when the optimizer can prove that no rows would be locked. [#124830][#124830]
+- Some privilege checks when scanning the [`crdb_internal.system_jobs`]({% link v24.1/crdb-internal.md %}) internal table now happen once before the scan, instead of once for each row. This will improve performance for queries that read from `crdb_internal.system_jobs`. [#117438][#117438]
+- Improved the initial range descriptor scan on startup. [Node startup]({% link v24.1/cockroach-start.md %}) should now be faster on larger stores [#123959][#123959]
```
Comment (rmloveland) [inline src/current/_includes/releases/v24.2/v24.2.0-alpha.1.md:97]: missing period at end of sentence

**Review by rmloveland — APPROVED** [review]
LGTM, had small optional comments - only serious comment is to remove the contributor list

**Review by rmloveland — APPROVED** [review]
Reviewed latest changes in [this commit](https://github.com/cockroachdb/docs/pull/18664/commits/97b8e5495abb98f0012d7e00c7d7953fc2e7a133) and LGTM!

**Pull request merged**